torque-gradle-plugin/build.gradle

```diff
@@ -7,7 +8,7 @@ repositories {
 dependencies {
     implementation gradleApi()
     implementation libraries.androidGradlePlugin
-    implementation project(internal.torque)
+    implementation project(':torque-core')
```

Comment:
Could you save `:torque-core` to a val like how it used to be `internal.torque` before splitting into two? Since it's also referenced in the torque-runner module now